swap/src/bin/swap.rs

```diff
@@ -152,7 +152,11 @@ async fn main() -> Result<()> {
 
             println!("{}", table);
         }
-
+        Command::Config => {
+            println!("Data Directory: {}", data_dir.display());
+            let config_json = serde_json::to_string_pretty(&env_config)?;
```

Comment:
Hang on why are we printing `env_config`?

Reply:
Is this command even any useful on the CLI side?

Reply:
How will the users know where the database is?

We set the env config. I think this command should print it otherwise it is just a command that prints the data dir

Reply:
But the env config doesn't contain the database path! The CLI doesn't have a config file, we need to compute the database path from the passed in data-dir and print it.